develop/dev-guide-sample-application-aws-lambda.md

```diff
@@ -0,0 +1,205 @@
+
+   - Endpoint Type is **Public**.
+   - Connect With is **General**.
+   - Operating System is your runtime environment.
+
+   <Tip>If you are running in Windows Subsystem for Linux (WSL), switch to the corresponding Linux distribution.</Tip>
+
+3. Click **Create Password** to generate a password.
+
+   <Tip>If you have generated a password before, you can use the original password directly, or click **Reset Password** to generate a new password.</Tip>
+
+4. Copy and paste the corresponding connection string to `env.json`. The following is an example:
+
+   ```json
+   {
+     "Parameters": {
+       "TIDB_HOST": "{gateway-region}.aws.tidbcloud.com",
+       "TIDB_PORT": "4000",
+       "TIDB_USER": "{prefix}.root",
+       "TIDB_PASSWORD": "{password}"
+     }
+   }
+   ```
+
+   Replace the placeholders in `{}` with the values obtained in the **Connect** window.
+
+5. [Configure Step.4 environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) in Lambda Function.
+
+</div>
+
+<div label="Self-hosted TiDB">
+
+1. Copy and paste the corresponding connection string. The following is an example:
+
+   ```json
+   {
+     "Parameters": {
+       "TIDB_HOST": "{tidb_server_host}",
+       "TIDB_PORT": "4000",
+       "TIDB_USER": "root",
+       "TIDB_PASSWORD": "{password}"
+     }
+   }
+   ```
+
+   Replace the placeholders in `{}` with the values obtained in the **Connect** window.
+
+2. [Configure Step.1 environment variables](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) in Lambda Function..
+
+</div>
+
+</SimpleTab>
+
+### Step 4: Run the sample application locally
+
+1. Install the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/install-sam-cli.html).
+
+2. Run the following command to start the application:
+
+    ```bash
+    sam local invoke --env-vars env.json -e event.json "tidbHelloWorldFunction"
+    ```
+
+3. Check the output in the terminal. If the output is similar to the following, the connection is successful:
+
+    ```bash
+    {"statusCode":200,"body":"{\"results\":[{\"Hello World\":\"Hello World\"}]}"}
+    ```
+
+## Key Code Snippets
+
+Complete code snippets are available in the [tidb-aws-lambda-quickstart](https://github.com/tidb-samples/tidb-aws-lambda-quickstart) GitHub repository.
+
+You can refer to the following key code snippets to complete your application development.
+
+### Connect to TiDB
+
+```typescript
+import mysql from 'mysql2';
+
+const pool = mysql.createPool({
+  host,
+  port,
+  user,
+  password,
+  database,
+  ssl: {
+    minVersion: 'TLSv1.2',
+    rejectUnauthorized: true,
+  },
+  waitForConnections: true,
+  connectionLimit: 1,
+  maxIdle: 1, // max idle connections, the default value is the same as `connectionLimit`
+  idleTimeout: 60000, // idle connections timeout, in milliseconds, the default value 60000
+  queueLimit: 0,
+  enableKeepAlive: true,
+  keepAliveInitialDelay: 0,
```

Comment:
Keep it simple, delete parameters with default values, It may make beginners feel confused.

```suggestion
  connectionLimit: 1,
  maxIdle: 1, // max idle connections, the default value is the same as `connectionLimit`
  enableKeepAlive: true
```